exercises/practice/armstrong-numbers/armstrong-numbers-tests.8th

```diff
@@ -1,11 +1,11 @@
-"Zero is an Armstrong number"    ( 0 armstrong?  ) true test_eq
-\ "Single digit numbers are Armstrong numbers"    ( 5 armstrong?  ) true test_eq
-\ "There are no 2 digit Armstrong Numbers"    ( 10 armstrong?  ) false test_eq
-\ "Three digit number that is not an Armstrong number"    ( 100 armstrong?  ) false test_eq
-\ "Three digit number that is an Armstrong number"    ( 153 armstrong?  ) true test_eq
-\ "Four digit number that is an Armstrong number"    ( 9474 armstrong?  ) true test_eq
-\ "Four digit number that is not an Armstrong number"    ( 9475 armstrong?  ) false test_eq
-\ "Seven digit number that is not an Armstrong number"    ( 9926314 armstrong?  ) false test_eq
-\ "Seven digit number that is an Armstrong number"    ( 9926315 armstrong?  ) true test_eq
+"Zero is an Armstrong number"    ( 0 armstrong?  ) test_true
+\ "Single digit numbers are Armstrong numbers"    ( 5 armstrong?  ) test_true
+\ "There are no 2 digit Armstrong Numbers"    ( 10 armstrong?  ) test_false
+\ "Three digit number that is not an Armstrong number"    ( 100 armstrong?  ) test_false
+\ "Three digit number that is an Armstrong number"    ( 153 armstrong?  ) test_true
+\ "Four digit number that is an Armstrong number"    ( 9474 armstrong?  ) test_true
+\ "Four digit number that is not an Armstrong number"    ( 9475 armstrong?  ) test_false
+\ "Seven digit number that is not an Armstrong number"    ( 9926314 armstrong?  ) test_false
+\ "Seven digit number that is an Armstrong number"    ( 9926315 armstrong?  ) test_true
 
 bye
```

Comment:
Missing last line, by definition.  This happens 18 times for this patch.

Each file that consists of lines, should have an EOL character for the last intended line.